x/lockup/genesis.go

```diff
@@ -9,9 +9,22 @@ import (
 // InitGenesis initializes the capability module's state from a provided genesis
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetLastLockID(ctx, genState.LastLockId)
+	for _, lock := range genState.Locks {
+		if err := k.ResetLock(ctx, lock); err != nil {
```

Comment:
Lets add a comment for what is ResetLock here

Reply:
Added comment